### DISCUSSION TITLE
Budibase like ErpNext?

### DISCUSSION THREAD

**Discussion by davidjeba** [Budibase/budibase#5034] [General]
Will Budibase available in industry specific templates ?

1.Resort Chain Industry
2.Self Storage Chain Industry
3.Airbnb like Industry etc?

a kickstarter kind of templates?

**Comment by joebudi** [comment]
Hey! We currently have no plans to build these templates specifically. But! We will allow users and our community to build templates, so there's a possibility they could build them. It's also important to note, our templates are internal purposes and will not include SaaS templates.
